resources/elbv1_load_balancers.go

```diff
@@ -0,0 +1,550 @@
+					},
+					{
+						Name:     "listener_load_balancer_port",
+						Type:     schema.TypeInt,
+						Resolver: schema.PathResolver("Listener.LoadBalancerPort"),
+					},
+					{
+						Name:     "listener_protocol",
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("Listener.Protocol"),
+					},
+					{
+						Name:     "listener_instance_protocol",
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("Listener.InstanceProtocol"),
+					},
+					{
+						Name:     "listener_ssl_certificate_id",
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("Listener.SSLCertificateId"),
+					},
+					{
+						Name: "policy_names",
+						Type: schema.TypeStringArray,
+					},
+				},
+			},
+			{
+				Name:     "aws_elbv1_load_balancer_policies_app_cookie_stickiness_policies",
+				Resolver: fetchElbv1LoadBalancerPoliciesAppCookieStickinessPolicies,
+				Columns: []schema.Column{
+					{
+						Name:     "load_balancer_id",
+						Type:     schema.TypeUUID,
+						Resolver: schema.ParentIdResolver,
+					},
+					{
+						Name: "cookie_name",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "policy_name",
+						Type: schema.TypeString,
+					},
+				},
+			},
+			{
+				Name:     "aws_elbv1_load_balancer_policies_lb_cookie_stickiness_policies",
+				Resolver: fetchElbv1LoadBalancerPoliciesLbCookieStickinessPolicies,
+				Columns: []schema.Column{
+					{
+						Name:     "load_balancer_id",
+						Type:     schema.TypeUUID,
+						Resolver: schema.ParentIdResolver,
+					},
+					{
+						Name: "cookie_expiration_period",
+						Type: schema.TypeBigInt,
+					},
+					{
+						Name: "policy_name",
+						Type: schema.TypeString,
+					},
+				},
+			},
+			{
+				Name:     "aws_elbv1_load_balancer_policies",
+				Resolver: fetchElbv1LoadBalancerPolicies,
+				Columns: []schema.Column{
+					{
+						Name:     "load_balancer_id",
+						Type:     schema.TypeUUID,
+						Resolver: schema.ParentIdResolver,
+					},
+					{
+						Name:     "policy_attribute_descriptions",
+						Type:     schema.TypeJSON,
+						Resolver: resolveElbv1loadBalancerPolicyPolicyAttributeDescriptions,
+					},
+					{
+						Name: "policy_name",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "policy_type_name",
+						Type: schema.TypeString,
+					},
+				},
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+func fetchElbv1LoadBalancers(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
+
+	c := meta.(*client.Client)
+	svc := c.Services().ELBv1
+	processLoadBalancers := func(loadBalancers []types.LoadBalancerDescription) error {
```

Comment:
why not move this into a function under elbv1_types and pass the svc + load balancers?